pkg/apis/core/doc.go

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 // +k8s:deepcopy-gen=package
+// +groupName=
```

Comment:
Another point: I really hesitate to make this another (non-empty) value. The whole PR is about being explicit about empty group name. If we turn this into a non-empty value, we reserve a group name again that somebody might want to use. Alternative `+groupName:empty=true` or some other marker. But this just adds complexity. Empty is empty string, let's not hide that 🙂 – also commented elsewhere: this case is really only happening in k/k because the legacy core group does not exist outside.